Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/Beckn/Search.hs

```diff
@@ -153,6 +157,16 @@ handler merchant sReq = do
 
   allFarePoliciesProduct <- getAllFarePoliciesProduct merchantId merchantOpCityId fromLocationLatLong toLocationLatLong
   let farePolicies = selectFarePolicy result.distance allFarePoliciesProduct.farePolicies
+  when (isNothing sReq.bapPhoneNum) $ do
+    throwError $ InternalError "Customer Phone Number is Null"
```

Comment:
So if we deploy bpp first, then  `sReq.bapPhoneNum` will be `Nothing` and all search requests will fail
And also all search requests will fail with E500 error if customer phone is not there.
Is it expected behavior?

Reply:
Changed the logic . If not found then cancellation dues will be 0